test_template.yml

```diff
@@ -34,10 +34,29 @@ jobs:
         with:
           packages: xdot rolldice
           version: ${{ github.run_id }}-${{ github.run_attempt }}
+      - id: check-additional-pkg-dir
+        run: |
+          # xdot requires python3-numpy so check that it was installed
+          echo "::set-output name=numpy-exists::$([ -d "/usr/lib/python3/dist-packages/numpy" ] && echo "true" || echo "false")"
+          echo "::set-output name=cached-pkgs::$(ls -1 ${cache_dir}/*.tar.gz | sort)"
+          echo "::set-output name=cached-pkg-count::$(ls -1 ${cache_dir}/*.tar.gz | wc -w)"
+        shell: bash
+      - id: check-cache-hash
+        run: |
+          # Generate and check the hash key and compare it against the one provided
+          value="rolldice=1.16-1build1 xdot=1.1-2 @ ${GITHUB_RUN_ID}-${GITHUB_RUN_ATTEMPT}"
```

Comment:
To make it consistent I am going to follow up by making it follow file convention.